updated dependencies

clarakosi opened the pull request:
Updated dependencies for swagger-router.

clarakosi committed:
updated dependencies

Pchelolo commented:
Something is wrong with it - there was only one test that has been run. Probably some mocha changes between version 3 and 5 caused that.

Pchelolo commented:
Note that during installation there's a warning from npm
```
npm WARN eslint-config-node-services@2.2.5 requires a peer of eslint-config-wikimedia@^0.5.0 but none was installed.
npm WARN eslint-config-node-services@2.2.5 requires a peer of eslint-plugin-jsdoc@^3.2.0 but none was installed.
npm WARN eslint-config-node-services@2.2.5 requires a peer of eslint-plugin-json@^1.2.0 but none was installed.
```

That's because the newer version of the services eslint configuration have some new peer dependencies (https://nodejs.org/en/blog/npm/peer-dependencies/) that should be explicitly specified in the package.json https://github.com/wikimedia/restbase/blob/master/package.json#L53-L56

That might be why only one test it completing.

d00rman reviewed (commented):
Given all the changes in the deps, it would be also good to bump the patch version of the package.

clarakosi committed:
updated dependencies and small change to a test

Removed .only from test in test/features/router
Updated patch number
Removed jsint and jscs

Pchelolo commented on package.json:
```diff
@@ -30,20 +30,23 @@
   "devDependencies": {
-    "mocha": "^3.1.0",
+    "mocha": "^5.2.0",
     "mocha-jshint": "^2.3.1",
```
Don't need to depend on it anymore.

clarakosi replied:
They are peer dependencies of `eslint-config-node-services` and without both eslint fails

Pchelolo replied:
I'm pretty sure that is not true https://github.com/wikimedia/eslint-config-node-services/blob/master/package.json#L26-L29

clarakosi replied:
sorry I was thinking of `eslint-plugin-jsdoc`

Pchelolo commented on package.json:
```diff
@@ -30,20 +30,23 @@
-    "coveralls": "^2.11.14",
+    "mocha-lcov-reporter": "^1.3.0",
+    "coveralls": "^3.0.2",
     "mocha-jscs": "^5.0.1",
```
Idem

Pchelolo reviewed (commented):
Can remove a couple of deps.

clarakosi committed:
removed jshint and jscs

Pchelolo commented:
+1

d00rman commented:
Tagged as [v0.7.2](https://github.com/wikimedia/swagger-router/tree/v0.7.2)
